views/components/BarGraphTooltip/index.jsx

```diff
@@ -1,38 +1,40 @@
-import React from 'react'
-import classnames from 'classnames'
-import { TOTAL_LABEL } from '../../../constants'
-import { isTargetShown, siteTooltipContent } from './helpers'
+import React, { Fragment } from 'react'
+import { SITE_NAMES } from '../../../server/utils/siteNames'
 
 import './BarGraphTooltip.css'
 
-const BarGraphTooltip = ({ active, payload, label, studyTotals }) => {
-  if (!active || !payload?.length) {
-    return null
-  }
+const BarGraphTooltip = ({
+  active,
+  payload,
+  label,
+  studyTotals,
+  displayLeft,
+  areMoreThanTenSites,
+}) => {
+  if (!active || !payload?.length) return null
 
   return (
     <div className="BarGraphTooltip">
-      <div className="BarGraphTooltip_header">
+      {displayLeft ? <div className="arrow-left" /> : null}
+      <div className="BarGraphTooltip_data">
         <div className="BarGraphTooltip_label">{label}</div>
-        <div className="BarGraphTooltip_value">
-          {isTargetShown(payload) ? 'Value / Target' : 'Value'}
-        </div>
-      </div>
-      {siteTooltipContent(payload, studyTotals[label]).map(
-        ({ labelColumn, valueColumn }) => {
+        <div className="BarGraphTooltip_values">Value</div>
+        {payload.map(({ name, value }) => {
           return (
-            <div
-              className={classnames('BarGraphTooltip_row', {
-                'BarGraphTooltip_row-total': labelColumn === TOTAL_LABEL,
-              })}
-              key={`${labelColumn}-${valueColumn}`}
-            >
-              <div className="BarGraphTooltip_label">{labelColumn}</div>
-              <div className="BarGraphTooltip_value">{valueColumn}</div>
-            </div>
+            <Fragment key={name}>
+              <div className="BarGraphTooltip_label">{name}</div>
+              <div className="BarGraphTooltip_values">{value}</div>
+            </Fragment>
           )
-        }
-      )}
+        })}
+        <div className="BarGraphTooltip_label">Total</div>
+        <div className="BarGraphTooltip_values">
+          {areMoreThanTenSites
```

Comment:
Why ten? Could this variable be named something more descriptive of its role within this component, like `showTotalsBySite`?

Reply:
renamed it be more direct
[f1fdf68](https://github.com/AMP-SCZ/dpdash/pull/683/commits/f1fdf681e593a6459f2d272d1dcee587aa88217c)

Reply:
I still see some references to `areMoreThanTenSites`. Also, I wonder if we can change this based on the bar width instead of a hard coded number of sites? 10 might work on the browser width you're looking at, but what about an ultra-wide monitor or a smaller screen?

Reply:
I made adjustments to this logic [678475a](https://github.com/AMP-SCZ/dpdash/pull/683/commits/678475a6d0f56b21d5a8665c540564c759c5a4bb)